assets/cart.js

```diff
@@ -49,16 +49,32 @@ class CartItems extends HTMLElement {
   }
```

Comment:
@scottgmeadows correct me if im wrong but i think we are removing the `x` on desktop https://screenshot.click/10-22-eii2o-cetb9.png

Reply:
In this case, it's fine with me because we're using the mobile treatment where the volume pricing info is behaving like a true popover requiring a click (instead of a tooltip that shows on hover).